src/nlv_object.h

```diff
@@ -87,9 +90,10 @@ class NLVObject : public NLVObjectBase
     return Nan::New(ParentClass::constructor_template)->HasInstance(val);
   }
 
-  HandleType handle() const {
-    return handle_;
-  }
+  const HandleType handle() const {
```

Comment:
as a purely informational refactor, do you think it would be a good idea to rename this to `virHandle()` so we avoid mental collisions with v8's `handle()` ?